python/cucim/src/cucim/core/operations/color/tests/test_color_jitter.py

```diff
@@ -0,0 +1,66 @@
+import cupy
+import numpy as np
+import pytest
+import skimage.data
+from PIL import Image
+
+import cucim.core.operations.color as ccl
+
+
+def get_image_array():
+    img = skimage.data.astronaut()
+    arr = np.asarray(img)
+    arr = np.transpose(arr)
+    return arr
+
+
+def verify_result(output, input):
+    diff_im = output - input
+    diff_total_value = np.abs(np.sum(diff_im))
+    assert diff_total_value >= 0
```

Comment:
For this would recommend using NumPy's [`allclose`]( https://numpy.org/doc/stable/reference/generated/numpy.allclose.html ). CuPy also implements this

Reply:
Color jitter randomly executes the sequence of 4 operations. It will be hard to determine the error tolerance here. Goal here is to test that the two arrays are different. We can update the code to fix the sequence of operations in future test cases.

Reply:
Ok let's leave it then